### ISSUE TITLE
Neighbourhood Generator and Cave/Glade Generator Add-on

### ISSUE THREAD

**Issue by MatthewBoyter** [TarkanAl-Kazily/one-page-parser#9]
Hi,  any chance of adding the other watabou generator maps to the importer?

**Comment by TarkanAl-Kazily** [comment]
Good idea, I assume you're thinking of the ones with floor plans (the recent mansion / building generator comes to my mind)?
